checkers/discover/generate.go

```diff
@@ -65,3 +65,95 @@ func GenerateAnalyzer(checkerDir, dest string) error {
 
 	return nil
 }
+
+// for file checkers/registry.go
+// modify this template when adding checkers for a new language
+var builtinCheckersGo = `// AUTOMATICALLY GENERATED: DO NOT EDIT
+
+package checkers
+
+import (
+	"globstar.dev/checkers/javascript"
+	"globstar.dev/checkers/python"
+	goAnalysis "globstar.dev/analysis"
+)
+
+type Analyzer struct {
+	TestDir   string
+	Analyzers []*goAnalysis.Analyzer
+}
```

Comment:
The type definition can be inside the checkers/checker.go file. The generated registry.go file need not contain this.